packages/flutter_tools/lib/src/runner/target_devices.dart

```diff
@@ -29,7 +29,7 @@ String _foundSpecifiedDevicesMessage(int count, String deviceId) =>
 String _noMatchingDeviceMessage(String deviceId) => 'No supported devices found with name or id '
     "matching '$deviceId'.";
 String flutterSpecifiedDeviceDevModeDisabled(String deviceName) => 'To use '
-    "'$deviceName' for development, enable Developer Mode in Settings → Privacy & Security.";
+    "'$deviceName' for development, open Xcode, reconnect the '$deviceName' and enable Developer Mode in Settings → Privacy & Security.";
```

Comment:
> Also see
> https://stackoverflow.com/questions/77648341/flutter-not-reading-device-developer-mode-setting

Thanks for the link! I can see now that this is a legitimate UX problem.

I think the text change proposed here is a little confusing (though it's possible this is due to me not testing on physical iOS devices very often). When I first read it, I thought it was telling me to change settings in Xcode, not on the device.

Referencing the StackOverflow thread linked, I propose this as a starting point:

```
To use <device> for development, enable Developer Mode in Settings → Privacy & Security on your device.

If this does not work, open Xcode, reconnect the device, and look for a popup on the device asking you to trust this computer.
```

Reply:
@andrewkolos I agree, your wording is better.

Reply:
Updated in commit [reword message](https://github.com/flutter/flutter/pull/154511/commits/99c01adee6d94a43b83f38b6e5f530ca621daac1)